runtime/doc/builtin.txt

```diff
@@ -3570,6 +3571,17 @@ getloclist({nr} [, {what}])				*getloclist()*
 			:echo getloclist(5, {'filewinid': 0})
 
 
+getmaps()						*getmaps()*
```

Comment:
All the map related functions start with the map prefix. To be consistent, this function should
be called mapget().

Reply:
Except hasmapto().  It would be nice to start with "map", but what to call it then?  mapgetall() ?  maplist()?
mapget() suggests getting one mapping.
I used getmappings() for now.

Reply:
Given the presence of `getloclist` I'd say `getmaplist` is the name that fits best the existing functions.